src/contrib/persistence/Akka.Persistence.Query.Sql/AllEventsPublisher.cs

```diff
@@ -20,16 +18,16 @@ internal static class AllEventsPublisher
         [Serializable]
         public sealed class Continue
         {
-            public static readonly Continue Instance = new Continue();
+            public static readonly Continue Instance = new();
 
             private Continue() { }
         }
 
-        public static Props Props(long fromOffset, TimeSpan? refreshInterval, int maxBufferSize, string writeJournalPluginId)
+        public static Props Props(long fromOffset, TimeSpan? refreshInterval, int maxBufferSize, IActorRef writeJournal, IActorRef queryPermitter)
```

Comment:
The `IActorRef queryPermitter` parameter has been added to all queries - this is the actor responsible for managing token buckets for permitting individual queries to run.